docs/book/src/user/quick-start.md

```diff
@@ -678,6 +678,23 @@ kubectl --kubeconfig=./capi-quickstart.kubeconfig get nodes
 {{#/tab }}
 {{#/tabs }}
 
+### Clean Up
+
+Delete workload cluster and wait for it to be completely cleaned up.
+```bash
+kubectl delete -f capi-quickstart.yaml
+```
+
+Delete infrastructure provider
+```bash
+clusterctl delete --infrastructure <infrastructure_provider>
```

Comment:
This is not necessary, given that we are nuking the kind cluster in the next step

Reply:
Thanks. Made the changes.